src/sklearn_evaluation/plot/pca.py

```diff
@@ -0,0 +1,150 @@
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+from ..telemetry import SKLearnEvaluationLogger
```

Comment:
Change this to : `from sklearn_evaluation.telemetry import SKLearnEvaluationLogger`